docs/promo.rst

```diff
@@ -134,46 +198,42 @@ cluster is being set up. Once it is done, this will show *Available* and you
 will be able to see cluster metrics.
 
 
-Discounted promotional cluster
-------------------------------
-
-In the case of a discounted - rather than free - cluster, the process is
-essentially the same as described above. Choose your discounted cluster in the
-subscription plan UI when you have clicked on *Create new subscription*, just
-as before. In the example used here, you could choose the discounted GP plan.
-
-.. image:: _assets/img/free-trial-plans.png
-   :alt: Choose a promoted subscription plan
+Free promotional cluster
+------------------------
 
-The only difference is that if you do not already have a means of payment
-associated with your organization, you will see an extra screen at the end of
-the cluster configuration wizard asking you to fill out your payment and
-billing information. (I.e., this appears as the fourth wizard screen, following
-step 3 described in the section above.) On this screen, you can fill out your
-credit card details and your billing address. We also ask you here to authorize
-payment to Crate.io AT GmbH for the costs of the (discounted) cluster.
+Just as clusters may be discounted based on subscription plan, tier, or number
+of nodes, so clusters may be offered free based on any of these dimensions. As
+with discounts, the free offer may last indefinitely or for a limited amount
+of time.
 
-.. image:: _assets/img/free-trial-wizard-step4.png
-   :alt: Promo cluster configuration wizard step 4
+In the case of a free cluster, the process is essentially the same as described
+above. Choose the promotional cluster in the cluster deployment screen when you
+have signed up and created an organization. When the cluster is promotional, it
+will have a % symbol under the node selector, as described above. Free
+clusters, as opposed to discounted ones, the price indicator bar at the bottom
+will show that it costs zero. If the cluster in question is free for a limited
+period of time, the price indicator bar will show what it will cost when it is
+no longer free, and after what period of time (in months) this will apply.
 
-After you have submitted this information, click *Next* to create the cluster
-as normal. The payment and billing information you have submitted will be saved
-in the Billing tab of the Organization overview page in the CrateDB Cloud
-Console.
+Naturally, for a free cluster, no billing information is required.
```

Comment:
Is it worth writing that billing information must be added before the promotion period expires, or otherwise the cluster will be deleted?

Reply:
Good point - it's sort of mentioned in the next section, but that could be made clearer, probably.